streams/src/main/java/org/apache/kafka/streams/processor/internals/TaskManager.java

```diff
@@ -692,12 +711,17 @@ Task taskForInputPartition(final TopicPartition partition) {
      * @return number of committed offsets, or -1 if we are in the middle of a rebalance and cannot commit
      */
     int commitAll() {
+        return commitAll(tasks.values());
+    }
+
+    private int commitAll(final Collection<Task> tasks) {
```

Comment:
This could be named as `commitInternal`

Reply:
+1, it could make the reading a bit easier.